app/pods/components/web-components/au-file-uploader/component.js

```diff
@@ -35,6 +37,7 @@ export default class FileUploader extends Component {
   }) *uploadFileTask(file) {
     try {
       this.isLoading = true;
+      this.showLoader = true;
```

Comment:
nergens een showLoader = false, maar toch verdwijnt die.
Willen we die ook niet in de finally ?